src/lib.rs

```diff
@@ -10,6 +10,7 @@
 //! the Trait.  The app doesn't do any actual processing on a transaction.
 //!
 //! ```rust,no_run
+//! #[derive(Clone, Copy)]
```

Comment:
shouldn't be needed

Reply:
Resolved in the version 
https://github.com/tendermint/rust-abci/pull/50/commits/f3f4653167f09b87e5fabb5f87ca4ee8e3425329